src/components/sbb-pearl-chain-vertical/sbb-pearl-chain-vertical.scss

```diff
@@ -0,0 +1,8 @@
+:host {
```

Comment:
Is there a specific reason why "@include host-component-properties;" was not declared here?